hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -144,20 +145,19 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       const dataSize = duResponse.size;
       let subpaths: IDUSubpath[] = duResponse.subPaths;
 
-      subpaths.sort((a, b) => (a.size < b.size) ? 1 : -1);
-
-      // Only show top n blocks with the most DU,
-      // other blocks are merged as a single block
-      if (subpaths.length > limit) {
+      // This logic is for other objects for Max Limit 30 we are calculating 
+      // Other Object Size = Total Size - Sum of all subpaths Size
```

Comment:
I don't understand this comment. I'd go with something like this:
```suggestion
      // We need to calculate the size of "Other objects" in two cases: 
      // 1) If we have more subpaths listed, than the limit.  
      // 2) If the limit is set to the maximum limit (30) and we have any number of subpaths. In this case we won't
      // necessarily have "Other objects", but later we check if the other objects's size is more than zero (we will have
      // other objects if there are more than 30 subpaths, but we can't check on that, as the response will always have 
      // 30 subpaths, but from the total size and the subpaths size we can calculate it).
```

Reply:
@dombizita 
Updated comment line.